operator-metadata/container.yaml

```diff
@@ -7,8 +7,8 @@ platforms:
   - aarch64
 operator_manifests:
   enable_digest_pinning: true
-  enable_repo_replacements: true
-  enable_registry_replacements: true
+  enable_repo_replacements: false
```

Comment:
what's this got to do with reverting [ENTMQST-6346](https://issues.redhat.com/browse/ENTMQST-6346)?

Reply:
@k-wall Nohing. On the call, we discussed three topics. Reverting the securityContext field, Kafka Exporter issue and getting the bundle image to build. @MikeEdgar suggested the changes in the `container.yaml` might fix the bundle issue so I added changes issues for both in the one PR, hoping to get a swift turnaround and getting the pipeline build running.